### PR TITLE
Update .drone.star and drop PHP 7.2

### PR DESCRIPTION
**Pull request by phil-davis**
Part of issue https://github.com/owncloud/core/issues/39134
and https://github.com/owncloud/QA/issues/687

### CODE REVIEW

File: .drone.star [hunk 1]
```diff
@@ -53,22 +44,7 @@ config = {
...
-                "webUITwoFactorTOTP": "webUITwoFactTOTP",
-            },
-            "browsers": [
-                "chrome",
-                "firefox",
-            ],
-            "servers": [
-                "daily-master-qa",
-            ],
-            "phpVersions": [
```
Comment (jasson99) [inline .drone.star:71]: Do we really want ot remove these?

File: .drone.star [hunk 2]
```diff
@@ -79,19 +55,7 @@ config = {
...
-        },
-        "webUIother-74": {
-            "suites": {
-                "webUIapiTwoFactorTOTP": "webUIapiTOTP",
-                "webUIcliTwoFactorTOTP": "webUIcliTOTP",
-            },
-            "servers": [
-                "daily-master-qa",
-            ],
-            "phpVersions": [
```
Comment (jasson99) [inline .drone.star:94]: this as well?

File: .drone.star (hunk 1 above)
Comment (phil-davis) [reply .drone.star:71]: It was running PHP 7.2 and 7.3 with the "default" servers (which is "latest" and "daily-master-qa") and also running PHP 7.4 with only "daily-master-qa".

I removed PHP 7.2. And put PHP 7,4 into the section that runs with the "default" serves. So actually more PHP 7,4 pipelines will run.

File: .drone.star (hunk 2 above)
Comment (phil-davis) [reply .drone.star:94]: Same as above - actually more PHP 7,4 pipelines should run.

**Review by jasson99 — APPROVED** [review]

**Pull request merged**